math/metis/Portfile

```diff
@@ -1,61 +1,41 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
 PortSystem          1.0
-PortGroup           cmake 1.0
-PortGroup           compilers 1.0
+PortGroup           cmake 1.1
+PortGroup           github 1.0
+PortGroup           active_variants 1.1
 
+github.setup        KarypisLab METIS 94c03a6e2d1860128c2d0675cbbb86ad4f261256
 name                metis
-version             5.1.0
-revision            3
+version             20220130
+epoch               1
```

Comment:
Best not to increase the `epoch` unnecessarily. It's not necessary here because the new `version` is greater than the old `version` as far as the `vercmp` procedure can see (20220130 > 5).

I recommend keeping a `revision 0` line, however.